test/mocha/test.js

```diff
@@ -0,0 +1,6 @@
+var assert = require('assert');
+describe('Mocha Test', function () {
```

Comment:
rename it to like Dummy Backend Test or something

Reply:
Dummy Backend Test as required